program/rust/src/tests/test_upd_price_v2.rs

```diff
@@ -32,7 +34,7 @@ use {
 };
 
 #[test]
-fn test_upd_price_v2() {
+fn test_upd_price_v2() -> Result<(), Box<dyn std::error::Error>> {
```

Comment:
I don't understand why you chose to change this test

Reply:
To use `?`. `assert!(result.is_ok())` will only tell you that it expected `true` and `false` was passed.